Resources/Private/Language/de.locallang.xlf

```diff
@@ -3,6 +3,10 @@
 	<file source-language="en" datatype="plaintext" original="messages" date="2013-03-40T10:38:40Z" product-name="events2">
 		<header/>
 		<body>
+			<trans-unit id="plugin.settings.moreLabel">
```

Comment:
I really like the idea, but we don't have much place in PageLayout view. Further I expect "Edit" to be a link to edit the categories, which it is not. Currently, I would prefer to use just ", ..." as ending.

Reply:
Done